packages/web-components/src/components/ic-breadcrumb-group/ic-breadcrumb-group.tsx

```diff
@@ -60,6 +60,14 @@ export class BreadcrumbGroup {
     }
   }
 
+  disconnectedCallback(): void {
+    this.breadcrumb.removeEventListener(
+      "transitionend",
+      this.transitionendHandler
+    );
+    this.renderCollapsedBreadcrumb("remove");
```

Comment:
think you could just remove the bound `clickHandler` here rather than call the above func, as calling that will creating element in the DOM